docs/CHANGELOG.rst

```diff
@@ -4,6 +4,19 @@ Change Log
 
 All notable changes to this project are documented in this file.
 
+==========
+Unreleased
+==========
+
+Changed
+-------
+
+Fixed
+-----
+
+Added
+-----
+* Added "flow_collection: sample" to processes
```

Comment:
@jkokosar, next time, use ``` ``flow_collection: sample`` ``` instead of ``` "flow_collection: sample" ``` so that Sphinx will render this as "code" (i.e. monospaced).